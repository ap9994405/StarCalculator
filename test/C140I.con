140
     1         2     4     8
     2         1     3    11
     3         2     5    14
     4         1     5     6
     5         3     4    17
     6         4     7    19
     7         6     9    22
     8         1     9    10
     9         7     8    25
    10         8    12    27
    11         2    12    13
    12        10    11    30
    13        11    15    32
    14         3    15    16
    15        13    14    35
    16        14    18    37
    17         5    18    20
    18        16    17    40
    19         6    20    21
    20        17    19    42
    21        19    23    44
    22         7    23    24
    23        21    22    46
    24        22    26    48
    25         9    26    28
    26        24    25    51
    27        10    28    29
    28        25    27    52
    29        27    31    54
    30        12    31    33
    31        29    30    57
    32        13    33    34
    33        30    32    58
    34        32    36    60
    35        15    36    38
    36        34    35    63
    37        16    38    39
    38        35    37    64
    39        37    41    66
    40        18    41    43
    41        39    40    69
    42        20    43    45
    43        40    42    70
    44        21    45    47
    45        42    44    72
    46        23    47    49
    47        44    46    74
    48        24    49    50
    49        46    48    76
    50        48    53    78
    51        26    52    53
    52        28    51    55
    53        50    51    81
    54        29    55    56
    55        52    54    82
    56        54    59    84
    57        31    58    59
    58        33    57    61
    59        56    57    87
    60        34    61    62
    61        58    60    88
    62        60    65    90
    63        36    64    65
    64        38    63    67
    65        62    63    93
    66        39    67    68
    67        64    66    94
    68        66    71    96
    69        41    70    71
    70        43    69    73
    71        68    69    99
    72        45    73    75
    73        70    72   100
    74        47    75    77
    75        72    74   102
    76        49    77    79
    77        74    76   104
    78        50    79    80
    79        76    78   106
    80        78    83   107
    81        53    82    83
    82        55    81    85
    83        80    81   110
    84        56    85    86
    85        82    84   111
    86        84    89   112
    87        59    88    89
    88        61    87    91
    89        86    87   116
    90        62    91    92
    91        88    90   117
    92        90    95   118
    93        65    94    95
    94        67    93    97
    95        92    93   122
    96        68    97    98
    97        94    96   123
    98        96   101   124
    99        71   100   101
   100        73    99   103
   101        98    99   128
   102        75   103   105
   103       100   102   129
   104        77   105   108
   105       102   104   130
   106        79   107   108
   107        80   106   109
   108       104   106   132
   109       107   113   133
   110        83   111   113
   111        85   110   112
   112        86   111   114
   113       109   110   115
   114       112   115   119
   115       113   114   136
   116        89   117   119
   117        91   116   118
   118        92   117   120
   119       114   116   121
   120       118   121   125
   121       119   120   137
   122        95   123   125
   123        97   122   124
   124        98   123   126
   125       120   122   127
   126       124   127   131
   127       125   126   138
   128       101   129   131
   129       103   128   130
   130       105   129   134
   131       126   128   135
   132       108   133   134
   133       109   132   139
   134       130   132   135
   135       131   134   140
   136       115   137   139
   137       121   136   138
   138       127   137   140
   139       133   136   140
   140       135   138   139
