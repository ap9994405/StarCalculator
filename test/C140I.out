 2178836352 + 17284258080 x + 63039569850 x^2 + 139959052860 x^3 + 211084339050 x^4 + 228507453624 x^5 + 182881210740 x^6 + 109783773300 x^7 + 49604283030 x^8 + 16763049900 x^9 + 4164183936 x^10 + 736246560 x^11 + 87462135 x^12 + 6242880 x^13 + 201960 x^14
 total: 1026080164257
